### PR TITLE
Add faxzero

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
**http://faxzero.com/**

**Send an online fax for free.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

**Comment by Anshuman-Verma** [comment]
I'm real bad at category selection. :disappointed:

File: README.md [hunk 1]
```diff
@@ -250,6 +250,7 @@ See [plotting graphs examples](https://www.wolframalpha.com/examples/PlottingAnd
 * [Web-Capture](https://web-capture.net/) - Webapp for converting webpages to popular formats(PDF, JPEG, ZIP) just by submitting link.
 * [PrintWhatYouLike](http://www.printwhatyoulike.com/) - Print any webpage or save it as HTML or PDF.
 * [WhatTheFont](http://www.myfonts.com/WhatTheFont/) - Determine font name from an image.
+* [FaxZero](https://faxzero.com/) - Send an online fax for free.
```
Comment (aviaryan) [inline README.md]: This can go in communication.

Comment (Anshuman-Verma) [inline README.md]: @aviaryan fixed

**Comment by Anshuman-Verma** [comment]
@aviaryan don't get exactly what's wrong with the comments.

**Comment by aviaryan** [comment]
@Anshuman-Verma I was talking about my review comment. 😅

**Pull request merged**